### ISSUE TITLE
[ISSUE] Build Config Error

### ISSUE THREAD

**Issue by MeCJay12** [WillyXJ/facileManager#446]
Replace everything between stars with current version of your facileManager and module installations:  
fM Version : 3.4.2
fmDNS Version : 3.3.3 and 3.3.4

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [X] I have read and understood the [contributors guide](https://github.com/WillyXJ/facileManager/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [X] I have checked that the issue I'm posting isn't already reported.
- [X] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/WillyXJ/facileManager/issues?q=is%3Aissue+is%3Aclosed) and [opened issues](https://github.com/WillyXJ/facileManager/issues)
- [X] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

---

**(BUG | ISSUE) Expected Behavior:**
Config built and pushed out to servers

**(BUG | ISSUE) Actual Behavior:**
![3 3 3](https://user-images.githubusercontent.com/43658705/58264620-2566ee80-7d4c-11e9-8f01-90e1dd87a1d7.PNG)
![3 3 4](https://user-images.githubusercontent.com/43658705/58264621-2566ee80-7d4c-11e9-95b6-0a15da8bd21d.PNG)

**(BUG | ISSUE) Steps to reproduce:**
Select servers, Action -> Build Config, then apply or change a record and process.

Connectivity tests in tools all pass.

**Comment by WillyXJ** [comment]
Is Bound1 or your fM server running Windows?

**Comment by MeCJay12** [comment]
No. My fM server is Bound and is running Ubuntu 16. All my DNS servers Bound1-3 are also Ubuntu 16. The client connecting to fM in this picture was the only thing Windows.

**Comment by WillyXJ** [comment]
Have you tried from a different browser and/or computer? This definitely looks client-related.

**Comment by MeCJay12** [comment]
Yup, that was it. I used Edge to build the config once and that fixed it in Chrome.
